DemoServer/Controllers/Demos/TextSearch/FTSQuerySearchOperators/metadata.json

```diff
@@ -0,0 +1,136 @@
+                    "Url"  : "https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/query-syntax-and-method-syntax-in-linq"
+                }
+            ]
+        },
+        {
+            "Title": "Define search criteria I",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* Use `Search()` to search for the specified terms on the specified document field.",
+                "",
+                "* The default operator between 'term1' and 'term2' is OR.<br>",
+                "  Use `@operator: SearchOperator.And` to set the logical AND operation instead.",
+
+                "* In this Search, we search for _term1 AND term2_ in the document field _Notes_."
+            ],
+            "Assets": [
+                {
+                    "Title": "How to search",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/client-api/session/querying/how-to-use-search"
+                },
+                {
+                    "Title": "Querying searching",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/querying/searching"
+                }
+            ]
+        },
+        {
+            "Title": "Define search criteria II",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* Another `Search()` method can be called for searching on another document field or terms.",
+                "",
+                "* The specified logical operation is applied between this 'Search' and the previous 'Search'.",
+                "",
+                "* With this Search added, we are now searching for ( _term1_ AND _term2_ ) OR _term3_ in the document field _Notes_."
+            ],
+            "Assets": [
+                {
+                    "Title": "How to search",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/client-api/session/querying/how-to-use-search"
+                },
+                {
+                    "Title": "Querying searching",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/querying/searching"
+                }
+            ]
+        },
+        {
+            "Title": "Execute the query",
+            "Slug": "step-4",
+            "DescriptionLines": [
+                "* Execute the query by calling `ToList()`.<br>",
+                "  An Auto-Index is created upon executing the query for the first time with 2 index fields:",
+                "    * _Notes_ - contains the original text from the indexed document field Notes.",
+                "",
+                "    * _Search(Notes)_ - contains terms from the Notes field that were tokenized by RavenDB's default analyzer (RavenStandardAnalyzer).<br>",
```

Comment:
Is it RavenStandardAnalyzer or StandardAnalyzer?

Reply:
done - changed to: RavenStandardAnalyze
(RavenStandardAnalyze encapsulates Lucene's StandardAnalyzer)